pkg/proxy/conntrack/conntrack.go

```diff
@@ -57,8 +59,12 @@ func newConntracker(handler netlinkHandler) Interface {
 }
 
 // ListEntries list all conntrack entries for connections of the given IP family.
-func (ct *conntracker) ListEntries(ipFamily uint8) ([]*netlink.ConntrackFlow, error) {
-	return ct.handler.ConntrackTableList(netlink.ConntrackTable, netlink.InetFamily(ipFamily))
+func (ct *conntracker) ListEntries(ipFamily uint8) (entries []*netlink.ConntrackFlow, err error) {
+	err = retry.OnError(util.MaxAttemptsEINTR, util.ShouldRetryOnEINTR, func() error {
+		entries, err = ct.handler.ConntrackTableList(netlink.ConntrackTable, netlink.InetFamily(ipFamily))
```

Comment:
I think you need to aggregate the entries here

Reply:
I had assumed that if it was a list call, that on retry it's possible to get the same set of entries back.
I'll see if I can test this behaviour, but it may be difficult to reproduce

Reply:
I'm struggling to reproduce this scenario, but I still stand by my assumption.
A call to list conntrack entries, after a return of a partial result, would return the entire list back. Unlike a delete, where once deleted, they wouldn't be returned.